lineman/app/components/group_page/group_help_card/group_help_card.haml

```diff
@@ -0,0 +1,17 @@
+%section.group-help-card
```

Comment:
Hmm... I wonder if we should display this only on loomio.org, and leave it off of future whitelabel versions?